modules/scripts/startup-script/files/running-script-warning.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+FINISH_LINE="startup-script exit status"
+ACTIVE=^activ
+WARNING=$(
+	cat <<EOF
+** WARNING **: The VM startup scripts for this have not been completed and
```

Comment:
I think this text is a bit verbose and maybe difficult for non-native English speakers. We should meet offline to make it shorter and align with Google documentation principles. e.g.

"It is advised that you wait..." can become "Wait...".